### PR TITLE
DB-7365 Check external table directory emptiness

### PR DESCRIPTION
**Pull request by changli6**

### CODE REVIEW

**Comment by dgomezferro** [comment]
@jyuanca did you mean to close the PR? I agree an IT would be needed.

**Comment by jyuanca** [comment]
@dgomezferro my mis-operation :-)

File: splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/CreateTableConstantOperation.java [hunk 1]
```diff
@@ -469,10 +469,10 @@ protected void createTable(Activation activation) throws StandardException {
                 DistributedFileSystem fileSystem = SIDriver.driver().getFileSystem(location);
                 FileInfo fileInfo = fileSystem.getInfo(location);
                 String[] files = fileSystem.getExistingFiles(location, "*");
-                if(!fileInfo.exists() || (files.length == 1 && files[0].equals("_SUCCESS"))) {
+                if(!fileInfo.exists() || (files.length == 0) || (files.length == 1 && files[0].equals("_SUCCESS"))) {
```
Comment (OlegMazurov) [inline splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/CreateTableConstantOperation.java]: Can you use ExternalTableUtils.isEmptyDirectory here as well (extend with an optional DistributedFileSystem parameter)?

Comment (changli6) [reply splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/actions/CreateTableConstantOperation.java]: The "fileInfo" was used several times in the code block, I'd like to keep it.  The other part of the IF condition used ExternalTableUtils.isEmptyDirectory.

**Pull request merged**